### ISSUE TITLE
Failure screen shots are blank

### ISSUE THREAD

**Issue by betelgeuse** [rubycdp/cuprite#119]
Whenever one of our system specs fails (Ruby on Rails 6) it generates a screen shot. They are in path tmp/screenshots/failures_r_spec_example_groups*

These screen shots are all blank.

**Comment by route** [comment]
@betelgeuse That's weird, could you please turn on debug logs and attach them here so I can take a look, I need browser version as well. Below is the code we use on CircleCI so that it attaches log when spec fails:

```
CUPRITE_LOGGER = StringIO.new
Capybara.register_driver :cuprite do |app|
  options = {
    process_timeout: 5,
    timeout: 10
  }
  options.merge!(logger: CUPRITE_LOGGER) if ENV['CI']

  driver = Capybara::Cuprite::Driver.new(app, options)
  process = driver.browser.process
  puts "Browser: #{process.browser_version}"
  puts "Protocol: #{process.protocol_version}"
  puts "V8: #{process.v8_version}"
  puts "Webkit: #{process.webkit_version}"
  driver
end

RSpec.configure do |config|
  config.around do |example|
    if ENV['CI']
      CUPRITE_LOGGER.truncate(0)
      CUPRITE_LOGGER.rewind
    end

    example.run

    if ENV['CI'] && example.exception && example.metadata[:js]
      if example.exception.message.frozen?
        raise CUPRITE_LOGGER.string
      else
        example.exception.message << "\n\nDebug info:\n" + CUPRITE_LOGGER.string
      end
    end
  end
end
```

**Comment by janklimo** [comment]
@betelgeuse are you on `rspec-rails` by any chance? This has only been fixed in `4.0.0` which came out at the end of March.

**Comment by betelgeuse** [comment]
> @betelgeuse are you on `rspec-rails` by any chance? This has only been fixed in `4.0.0` which came out at the end of March.

I am so this could be the reason. I will report what happens once we get around to upgrading.

**Comment by route** [comment]
Please reopen if upgrading didn't help
